chart/skywalking/templates/es-init.job.yaml

```
@@ -15,7 +15,7 @@
 
 # https://docs.sentry.io/server/installation/docker/#running-migrations
 
-{{- if contains "elasticsearch" .Values.oap.storageType }}
+{{- if and .Values.oap.initEs (contains "elasticsearch" .Values.oap.storageType) -}}
```

The CI failed perhaps because of this (not sure tho)

```suggestion
{{- if and .Values.oap.initEs (contains "elasticsearch" .Values.oap.storageType) }}
```